pycloudlib/oci/cloud.py

```diff
@@ -21,7 +21,7 @@ class OCI(BaseCloud):
 
     def __init__(
         self, tag, timestamp_suffix=True, compartment_id=None,
-        config_path='~/.oci/config',
+        availability_domain=None, config_path='~/.oci/config',
```

Comment:
I know this the default value here is probably to maintain backwards compatibility, but since this a necessary parameter now I think we should not have a default value for it.

Also, I think we should also update the `oci` example file to include that change as well

Reply:
Yeah, I have always been of the opinion that the common arguments between the clouds (tag and timestamp_suffix) should go first in `__init__`. Since timestamp_suffix has a default, everything after it would also have to have a default, including "required" arguments.

I know we already have another cloud or two that breaks this, but I was hoping to change it to be consistent, not the other way around. I fine doing whatever the team thinks is best though.

Thanks for the reminder about the example

Reply:
That's a good point about the consistency. Fair enough. I don't see a problem keeping it as it is to enforce that